charts/soroban-rpc/templates/soroban-rpc-sts.yaml

```diff
@@ -43,19 +46,38 @@ spec:
       containers:
       - name: soroban-rpc
         image: {{ include "common.sorobanRpcImage" . | quote }}
-        command:
-        - /bin/bash
-        - -c
-        # sleep command in the following command is temporary arrangement since soroban rpc needs everything in place. 
-        - sleep 10 && /usr/bin/stellar-soroban-rpc --endpoint=0.0.0.0:{{.Values.sorobanRpc.sorobanRpcConfig.port | default 8000 }}  --log-level={{.Values.sorobanRpc.logLevel | default  "info" | quote}} --stellar-core-url={{ include "common.stellarCoreUrl" .}} --network-passphrase={{.Values.sorobanRpc.networkPassphrase | default "Test SDF Future Network ; October 2022" | quote}}
+        args: ["--config-path", "/config/soroban-rpc.cfg"]
         imagePullPolicy: {{ .Values.global.image.sorobanRpc.pullPolicy }}
+        volumeMounts:
+        - name: config-volume
+          mountPath: /config
+          readOnly: true
+        {{- if .Values.sorobanRpc.persistence.enabled }}
+        - name: {{ template "common.fullname" . }}-var-lib-stellar
```

Comment:
I'm not sure on rpc restart behavior with prior data such as from the pv, will be interesting  to test restarts or upgrades/new deploys of the stateful service after rpc has been running for a couple hours, and see how that goes, if captive core can deal with the prior bucket files and rpc with the sqlitedb, it does look like rpc has db migrations capability so it could upgrade the db if that were triggered.

Reply:
yes, early and often sounds good, per your suggestion, using beta versions on chart yaml for now like `version: 1.0.0-alpha.1` should give time to do iterations on testing and inform others it's not GA yet.

Reply:
Good idea, I updated version number with your suggestion.